### PR TITLE
OHRM5X-2395: AWS CLI Addition

### PR DESCRIPTION
**Pull request by devishke-orange**

### CODE REVIEW

File: aws/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+This tool assists AWS Marketplace Subscribers with managing their installation of OrangeHRM 5.x
+
+The commands have been grouped into three main categories:
+- OrangeHRM Commands: General commands for managing your OrangeHRM instance
+- OrangeHRM Backup Commands: Commands for  managing backups of your OrangeHRM instance
+- OrangeHRM SSL Commands: Commands for generating SSL certificates for your OrangeHRM instance
+
+## User Guide
+
+Our user guide for setting up your EC2 instance with the OrangeHRM AMI can be found [here](https://docs.google.com/document/d/1E4mSuN21M54IHjWaXx9DNE1SEtm9EEvj9QWffeVHbm4/edit?usp=sharing). This README will only provide list the available commands.
```
Comment (RajithaKumara) [inline aws/README.md]: change to help portal link

Comment (devishke-orange) [inline aws/README.md]: Change to help portal link

File: aws/compose.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+services:
+  orangehrm:
+    image: orangehrm/orangehrm:5.5
```
Comment (RajithaKumara) [inline aws/compose.yml:3]:
```suggestion
    image: orangehrm/orangehrm
```
`latest`

File: aws/assets/license.txt [hunk 3]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```
Comment (RajithaKumara) [inline aws/assets/license.txt]: we may have to change the license for 5.6

File: aws/assets/ssl.conf [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<IfModule mod_ssl.c>
```
Comment (RajithaKumara) [inline aws/assets/ssl.conf:1]: https://gist.github.com/RajithaKumara/87a69a196691083829ce4057dac47bdc#file-orangehrm-recommended-server-config-md
`X-frame-options` need to consider

File: aws/orangehrm [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+
+# Docker container and volume names
+export ORANGEHRM_CONTAINER="orangehrm"
+export MARIADB_CONTAINER="mariadb"
+export CERTBOT_CONTAINER="certbot"
+export ORANGEHRM_WEBROOT_VOLUME="orangehrm_orangehrm-webroot"
+export ORANGEHRM_DB_VOLUME="orangehrm_orangehrm-db"
+
+# Update URL
+export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/os-cloud/master/aws/compose.yml"
```
Comment (RajithaKumara) [inline aws/orangehrm]:
```suggestion
export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/os-cloud/main/aws/compose.yml"
```

File: aws/tests/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+orangehrm install
+```
+3. Login and change the password
+4. Install git in the AMI
+```
+sudo dnf install git
+```
+5. Run script to setup tests
+```
+curl -o- https://raw.githubusercontent.com/devishke-orange/orangehrm-aws/main/tests/setup_tests.sh | bash
```
Comment (RajithaKumara) [inline aws/tests/README.md]: TODO

**Review by RajithaKumara — APPROVED** [review]

File: aws/README.md (hunk 1 above)
Comment (devishke-orange) [reply aws/README.md]: changed

Comment (devishke-orange) [reply aws/README.md]: changed

File: aws/compose.yml (hunk 2 above)
Comment (devishke-orange) [reply aws/compose.yml:3]: did not change this since version checking happens from here

File: aws/assets/license.txt (hunk 3 above)
Comment (devishke-orange) [reply aws/assets/license.txt]: new license added

File: aws/orangehrm (hunk 5 above)
Comment (devishke-orange) [reply aws/orangehrm]: changed to `main`

File: aws/tests/README.md (hunk 6 above)
Comment (devishke-orange) [reply aws/tests/README.md]: changed link

**Pull request merged**